tidb-cloud/data-service-get-started.md

```diff
@@ -0,0 +1,216 @@
+
+This document introduces how to quickly get started with TiDB Cloud Data Service (beta) by creating a Data App, developing, testing, publishing, and calling an endpoint.
+
+## Before you begin
+
+Before creating a Data App, make sure that you have created a [Serverless Tier](/tidb-cloud/serverless-cluster.md) cluster. If you do not have one, follow the steps in [Create a cluster](/tidb-cloud/create-a-cluster.md).
+
+## Step 1. Create a Data App
+
+The Data App is a group of endpoints that you can use to access data for a specific application. To create a Data App, perform the following steps:
+
+1. Log in to the [TiDB Cloud console](https://tidbcloud.com).
+
+2. In the left navigation pane, click <svg width="16" height="16" viewBox="0 -2 20 20" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M15.8333 7.50002C15.8333 10.7217 13.2217 13.3334 10 13.3334M15.8333 7.50002C15.8333 4.27836 13.2217 1.66669 10 1.66669M15.8333 7.50002H4.16667M10 13.3334C6.77834 13.3334 4.16667 10.7217 4.16667 7.50002M10 13.3334C11.4591 11.736 12.289 9.663 12.3341 7.50002C12.289 5.33704 11.4591 3.26406 10 1.66669M10 13.3334C8.54092 11.736 7.71249 9.663 7.66743 7.50002C7.71249 5.33704 8.54092 3.26406 10 1.66669M10 13.3334V15M4.16667 7.50002C4.16667 4.27836 6.77834 1.66669 10 1.66669M11.6667 16.6667C11.6667 17.5872 10.9205 18.3334 10 18.3334C9.07953 18.3334 8.33333 17.5872 8.33333 16.6667M11.6667 16.6667C11.6667 15.7462 10.9205 15 10 15M11.6667 16.6667H17.5M8.33333 16.6667C8.33333 15.7462 9.07953 15 10 15M8.33333 16.6667H2.5" stroke="#333333" stroke-width="1.5" stroke-linecap="round" stroke-linejoin="round"/></svg> **Data Service**.
+
+3. On the **Get started by creating your first data application** page, enter a name for the Data App in the **Data App Name** field and click **Create Service**. The [**Data Service**](https://tidbcloud.com/console/dataservice) details page is displayed.
+
+## Step 2. Develop an endpoint
+
+An endpoint is a web API that you can customize to execute SQL statements.
+
+After creating a Data App, a default endpoint `New Endpoint` is created for you automatically. You can use the default endpoint to access your TiDB Cloud cluster.
+
+If you want to create a new endpoint, locate the newly created Data App and click **+** **Create Endpoint** on the top of the left pane.
+
+### Configure properties
+
+On the right pane, click the **Properties** tab and set properties for the endpoint, such as:
+
+- **Path**: the unique path of the endpoint that users use to access it. The path should be unique within a Data App and is appended to the service URL of the endpoint.
+
+    - To obtain the service URL of the endpoint, hover your mouse over **SERVICE_URL**. For example, `https://data.tidbcloud.com/api/v1beta/apps/{APP_ID}/endpoint`.
+    - Only letters, numbers, and underscores (`_`) are allowed in the path, which must start with a slash (`/`). For example, `/my_endpoint`.
```

Comment:
The path must be a maximum of 64 characters in length and can contain letters, digits, underscores (_), forward slashes (/) and must start with a forward slash (/)